README.md

```diff
@@ -0,0 +1,57 @@
+# ami-builder-configs
+Configuration files for building various AMIs using ami-builder
+
+Each time ami-builder-config code has a new commit & is merged, a new GitHub release is created and zipped
+This gets picked up by Concourse pipeline job 'Ami-builder-release' which is monitoring every hour looking for a new release.
+The Concourse pipeline order can be viewed in the docs folder for this repo.
+The reason this is done this way is you can't have push notifications from GitHub into Crown and there are rate limits for polling GitHub
+The Concourse pipeline job 'Upload-ami-buider-configs’ un-packs the new release and uploads the (config) files to an S3 bucket.
+Inside the bucket there is a Kafka folder and a DKS folder respectively.
```

Comment:
I'd probably re-word this for clarity to something like "Inside the bucket there is a folder for each specific AMI - currently there are Kafka and DKS AMIs". If that's right of course :)